pkgs/development/libraries/fontconfig/make-fonts-conf.nix

```
@@ -1,5 +1,11 @@
-{ runCommand, stdenv, lib, libxslt, fontconfig, dejavu_fonts, fontDirectories
-, impureFontDirectories ? [
+{ runCommand, stdenv, lib, libxslt, fontconfig, dejavu_fonts }:
+
+{
+  # an array of fonts, e.g. `[ pkgs.dejavu_fonts.minimal ]`
+
+  fontconfig ? fontconfig
```

My bad, I didn't know this would cause infinite recursion. This is more plausible.
```suggestion
let fontconfig_ = fontconfig; in
{
  fontconfig ? fontconfig_
  # an array of fonts, e.g. `[ pkgs.dejavu_fonts.minimal ]`
```